README.md

```diff
@@ -0,0 +1,103 @@
+## Generate a new Maven project ##
+
+- Run below command from Command line tool
+
+```text
+mvn archetype:generate -DartifactId=selenium-learning -DgroupId=com.mytest -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4
+```
+
+- Run below command to download Maven Dependencies, compile, and build project
+
+```text
+mvn clean package -e
+```
+OR,
+
+```text
+mvn clean install -e
+```
+**NOTE:** Use argument `-e` to console error messages while building the project 
+
+- Run below command to run unit tests against compile source code using available testing framework
+
+```text
+mvn test
+```
+
+## Add required dependency for Maven project ##
+
+- Update/Modify the deppendencies in `pom.xml`. Refer [Maven Central Repository](https://central.sonatype.com/) to know about the latest version of dependencies.
+
+1. Add Selenium Dependency
+
+```xml
+<dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>
+      <version>4.18.1</version>
+</dependency>
+```
+2. Add Dependendency to load and manage Browsers
+
+```xml
+<dependency>
+      <groupId>io.github.bonigarcia</groupId>
+      <artifactId>webdrivermanager</artifactId>
+      <version>5.7.0</version>
```

Comment:
**Suggestion**: Consider updating the version of the WebDriverManager dependency to the latest stable release for enhanced functionality and compatibility. <br> **Code Suggestion**: 
 ```
 +      <version>5.7.1</version>
 ```